conans/client/generators/cmake.py

```diff
@@ -32,6 +32,19 @@ def join_defines(values, prefix=""):
         self.src_paths = join_paths(cpp_info.src_paths)
 
         self.libs = join_flags(" ", cpp_info.libs)
+
```

Comment:
What about the targets approach and the find_package and find_package_multi approaches? I assume the target should contain also the frameworks?

Reply:
I've tested target approach (`CONAN_PKG:xxx`), it worked for me
other cmake generators, like `cmake_fine_package` or `cmake_find_package_multi`, I didn't test at all

Reply:
You have to. We cannot release a new attribute in the cppinfo if the generators are not working

Reply:
tested all 4 approaches:
- `CONAN_LIBS`
- `CONAN_PKG::`
- `cmake_find_package`
- `cmake_find_package_multi`
https://github.com/bincrafters/conan-sfml/commit/1e07f5b25536e2f787e2db083ab5bb6ea5488a27